linera-execution/src/lib.rs

```diff
@@ -466,6 +475,41 @@ pub struct MessageContext {
     pub message_id: MessageId,
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct ProcessStreamsContext {
+    /// The current chain ID.
+    pub chain_id: ChainId,
+    /// The authenticated signer of the operation that created the message, if any.
+    #[debug(skip_if = Option::is_none)]
+    pub authenticated_signer: Option<AccountOwner>,
```

Comment:
I believe this should always be `None` -- since stream updates are automatically processed (like incoming messages) by standard clients.